WordPress/src/main/java/org/wordpress/android/ui/posts/PostListFeaturedImageTracker.kt

```diff
@@ -47,6 +66,9 @@ class PostListFeaturedImageTracker(private val dispatcher: Dispatcher, private v
     }
 
     fun invalidateFeaturedMedia(featuredImageIds: List<Long>) {
-        featuredImageIds.forEach { featuredImageMap.remove(it) }
+        featuredImageIds.forEach {
+            featuredImageMap.remove(it)
+            ongoingRequests.remove(it)
+        }
```

Comment:
Will this be called when the featured image request is completed? I mention this because I haven't found any other place where the ongoing request is removed.

Reply:
Hey @fluiddot 
The places where this funcation gets called is `PostListEventListener` → `featuredMediaChanged`. 
The `featuredMediaChanged` is called when the media is changed or uploaded. Correct me if I am wrong. @zwarm

Reply:
Thanks for the question, which @AjeshRPai answered before me down below https://github.com/wordpress-mobile/WordPress-Android/pull/20939#discussion_r1630880715

Reply:
> The places where this funcation gets called is PostListEventListener → featuredMediaChanged.
The featuredMediaChanged is called when the media is changed or uploaded. Correct me if I am wrong.

This is correct, previously only success calls were removed from the featureImageMap and in the updated version, both success and errors are removed. We need to make sure we don't have requests hanging around when they shouldn't be.